base/inc/fields/siteorigin-widget-field-class-loader.class.php

```diff
@@ -64,6 +72,8 @@ public function load_field_class( $field_classname ) {
 
 		foreach( $this->class_paths as $class_path ) {
 			$filepath = $class_path . $filename . '.class.php';
+			// TODO: Fix. Currently, if there is more than one file with the same name in the different class paths,
+			//they will all be loaded. Shouldn't cause any problems, just not as efficient as it could be.
```

Comment:
Yeah, that should do it. My brain was focussed on other things, so I just made a note and hit a `continue`.